### ISSUE TITLE
Any chance this project can test build with JBR21?

### ISSUE THREAD

**Issue by hmy65** [amir1376/ab-download-manager#230]
Do you have problem with maximise/minimise window animation when build with jbr21?
I use reflection to customize compose window titlebar, I build with jbr17 like this project. I tried JBR21 but has no amimation.
Do you face same problem, cuz I saw you CustomWindowDecorationAccessing as well.

**Comment by hmy65** [comment]
@amir1376

**Comment by hmy65** [comment]
I tested on Mac/Win, both have no chance.

**Comment by amir1376** [comment]
Hi there. yes custom window decoration api has changed in JBR21
and it needs to re implemented in our side.
if you want to implement something like this but with jbr 21 you can check the [Jetbrains Jewel library](https://github.com/JetBrains/jewel)

**Comment by hmy65** [comment]
I tried Jewel as well. But there are three deal breakers.
1. Dragging on the TitleBar to move the window does not always work #368 https://github.com/JetBrains/jewel/issues/368  
2. The jewel title bar is always a gradient color, can't change to static color based on light/dark theme unless fork the entire project and change private api.
3. Proguard can't work with jewel, I wasted 2 hours still no luck:(
